modules/docs/mdx/12.0-UPGRADE-GUIDE.mdx

```diff
@@ -69,27 +70,35 @@ in a future major release. This signals consumers to migrate to a more stable al
 deprecated code is removed.
 
 ## Removals
-Removals are deletions from our codebase and you can no longer consume this component. We've either promoted or replaced a component or utility.
+
+Removals are deletions from our codebase and you can no longer consume this component. We've either
+promoted or replaced a component or utility.
 
 ### Select (Preview)
 
 **PR:** [#2796](https://github.com/Workday/canvas-kit/pull/2796)
 
-We've removed the `Select` component that was in Preview. Please use the compound [`Select` component from Main](https://workday.github.io/canvas-kit/?path=/docs/components-inputs-select--basic).
+We've removed the `Select` component that was in Preview. Please use the compound
+[`Select` component from Main](https://workday.github.io/canvas-kit/?path=/docs/components-inputs-select--basic).
 
 ---
 
 ## Component Updates
 
 ### Styling API and CSS Tokens
 
+**PRs:** [#2825](https://github.com/Workday/canvas-kit/pull/2825),
+[#2795](https://github.com/Workday/canvas-kit/pull/2795)
+
 Several components have been refactored to use our new
 [Canvas Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) and
 [styling API](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
 The React interface **has not changed**, but CSS variables are now used for dynamic properties.
 
 The following components are affected:
 
+- `TextInput`
+- `TextArea`
```

Comment:
Super NIT: This should be alphabetized and I can update it in a random PR if needed.